### PR TITLE
Feature/#6

### PR DESCRIPTION
**Pull request by nikolinat**

### CODE REVIEW

File: src/main/java/com/kti/restaurant/controller/OrderController.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        return new ResponseEntity<>(order, HttpStatus.OK);
+    }
+
+    @RequestMapping(value="/{id}", method=RequestMethod.DELETE)
+    public ResponseEntity<Order> deleteOrder(@PathVariable("id") Integer id) throws Exception {
+        orderService.delete(id);
+        return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+    }
+
+    @PutMapping("")
```
Comment (jojev) [inline src/main/java/com/kti/restaurant/controller/OrderController.java:58]: We need to send the id in the path.

File: src/main/java/com/kti/restaurant/mapper/OrderItemMapper.java [hunk 2]
```diff
@@ -1,30 +1,60 @@
...
+
+    public OrderItemDto fromOrderItemToOrderItemDto(OrderItem orderItem) {
+        return new OrderItemDto(orderItem.getId(), orderItem.getQuantity(), orderItem.getNote(),
+                orderItem.getStatus(), orderItem.getPriority(), orderItem.getOrder().getId());
+    }
+
+    private MenuItem findMenuItemById(Integer id) throws Exception {
+        MenuItem menuItem = menuItemService.findById(id);
+
+        if(menuItem == null){
```
Comment (jojev) [inline src/main/java/com/kti/restaurant/mapper/OrderItemMapper.java:44]: This validation exists in the MenuItemService so not necessary here.

File: src/main/java/com/kti/restaurant/mapper/OrderItemMapper.java [hunk 3]
```diff
@@ -1,30 +1,60 @@
...
+            throw new MissingEntityException("The menu item with given id does not exist in the system.");
+        }
+
+        return menuItem;
+    }
+
+    private Order findOrderById(Integer id) throws Exception {
+        Order order = orderService.findById(id);
+
+        if(order == null){
```
Comment (jojev) [inline src/main/java/com/kti/restaurant/mapper/OrderItemMapper.java:54]: This validation exists in the OrderService so not necessary here.

File: src/main/java/com/kti/restaurant/mapper/OrderMapper.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    public Order fromUpdateOrderDtoToOrder(UpdateOrderDto orderDto) {
+        return new Order(orderDto.getId(), orderDto.getStatus(),orderDto.getDateOfOrder(),orderDto.getPrice());
+    }
+
+    private RestaurantTable findRestaurantTableById(Integer id) throws Exception {
+        RestaurantTable restaurantTable = restaurantTableService.findById(id);
+
+        if(restaurantTable == null){
```
Comment (jojev) [inline src/main/java/com/kti/restaurant/mapper/OrderMapper.java:33]: Same as in the previous two comments.

**Review by jojev — APPROVED** [review]
I would delete those validations and everything else is okay.

**Review by draganaF — APPROVED** [review]

**Review by vAndrija — APPROVED** [review]

**Pull request merged**